### PR TITLE
issue #5228 - Fix custom theme usage

### PR DESCRIPTION
**Pull request by MV88**
## Description

Fix custom theme usage

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [x] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [ ] Other... Please describe:

## Issue

**What is the current behavior?**

#5528 

**What is the new behavior?**

now custom theme are correctly loaded for project aligned to 2020.02.xx 

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

**Comment by MV88** [comment]
> I didn't get why this is a regression and the real reason.

The main reason is that to the Theme component was not being passed a correct theme property hence the value default would have been used.

this fixes the previous behaviour that was passing a custom theme name down the tree

File: web/client/components/app/StandardRouter.jsx [hunk 1]
```diff
@@ -17,10 +17,13 @@ import Localized from '../I18N/Localized';
 import Theme from '../theme/Theme';
 import {ErrorBoundary} from 'react-error-boundary';
 import ErrorBoundaryFallbackComponent from './ErrorFallBackComp';
+import assign from 'object-assign';
 
 const ThemeProvider = connect((state) => ({
     theme: state.theme?.selectedTheme?.id
-}))(Theme);
+}), {}, (stateProps, dispatchProps, ownProps) => {
```
Comment (mbarto) [inline web/client/components/app/StandardRouter.jsx:24]: Does this make theme in ownProps win over the state? Did I understood the fix correctly?

Comment (MV88) [reply web/client/components/app/StandardRouter.jsx:24]: yes

Comment (mbarto) [reply web/client/components/app/StandardRouter.jsx:24]: good for me, then

**Comment by offtherailz** [comment]
Please backport to 2020.02.xx

**Pull request merged**